Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs

```diff
@@ -133,8 +141,7 @@ void IActivate.Activate(ActivateEventArgs args)
             var activeHandEntity = hands.GetActiveHand?.Owner;
             if (activeHandEntity == null)
             {
-                UpdateUserInterface();
-                UserInterface?.Open(actor.playerSession);
+                OpenUserInterface(actor);
```

Comment:
Any way to consolidate this with the above? The code looks the same.

Reply:
What should I try to consolidate this line / function with?